lib/hammer_cli/output/dsl.rb

```diff
@@ -20,6 +20,7 @@ def build(&block)
     def field(key, label, type=nil, options={}, &block)
       options[:path] = current_path.clone
       options[:path] << key if !key.nil?
+      options[:field_id] = options[:field_id] || key || options[:key]
```

Comment:
This is inconsistent with the Field class defaults. If field is created directly without the `:field_id` the default id is 'label'. Via dsl the 'key' wins. What about not setting the default value in here and leave it up to Field class?